ports/starlink-ast/portfile.cmake

```
@@ -0,0 +1,56 @@
+vcpkg_fail_port_install(ON_TARGET "Linux" "OSX")
+
+vcpkg_download_distfile(ARCHIVE
+    URLS "https://github.com/Starlink/ast/releases/download/v9.2.4/ast-9.2.4.tar.gz"
+    FILENAME "ast-9.2.4.tar.gz"
+    SHA512 84e6f243e6d9d77328b73b97355feb3990307fb9c8f9b2f30344d71e2f5e63a849cdce0090ff5b7cc16028e12d68516c885b13d76db841072c9d1d06a7742a9e
+)
+
+vcpkg_extract_source_archive_ex(
+    OUT_SOURCE_PATH SOURCE_PATH
+    ARCHIVE ${ARCHIVE}    
+)

```

```suggestion
vcpkg_from_github(
    OUT_SOURCE_PATH SOURCE_PATH
    REPO Starlink/ast
    REF v9.2.4
    SHA512 84e6f243e6d9d77328b73b97355feb3990307fb9c8f9b2f30344d71e2f5e63a849cdce0090ff5b7cc16028e12d68516c885b13d76db841072c9d1d06a7742a9e
    HEAD_REF master
)
```

Or is there is reason to download the tar.gz file?

---

the tar.gz file is the official release. The repo seems to be missing some folders and files that are generated by the package author